expo/tests/gbe/test_bid_class.py

```diff
@@ -4,10 +4,13 @@
 from django.test.client import RequestFactory
 from django.test import Client
 from gbe.views import bid_class
-import factories
-from functions import (
-    location,
-)
+import mock
+import gbe.ticketing_idd_interface 
+from tests.factories import gbe_factories as factories
+from tests.functions.gbe_functions import (login_as,
```

Comment:
Also, line break after paren to avoid pep-8 problems and make the style consistent.